doc/jobs/prerelease_jobs.rst

```diff
@@ -101,3 +112,32 @@ scripts which can be invoked manually in the same order:
   workspace.
   It also invokes the tool ``catkin_test_results --all`` to output a
   summary of all tests in the overlay workspace.
+
+Run the *prerelease* job on Travis
+----------------------------------
+
+Since it is easy to run a *prerelease* job locally it can also be run on Travis to either test every commit or pull request.
+The setup and invocation is the same as locally.
+The `.travis.yml <https://github.com/ros-infrastructure/ros_buildfarm/blob/master/.travis.yml>`_ file of the *ros_buildfarm* repository is a good starting point.
+
+Run for "custom" repositories
+-----------------------------
+
+Most job types require that the tested repository is being listed in a ROS distribution file.
+For the *prerelease* job that is not necessary.
```

Comment:
The feature is also not new but has existed since version 0.1.2 (9ac45b3edceb06574dd609548e3dce7b1f19597e).